docs/debug/2_config_file_structure.ipynb

```diff
@@ -0,0 +1,241 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Config File Structure\n",
+    "\n",
+    "To enable debug features, create a `config.yaml` file to specify the desired behavior, such as determining which GEMMs (General Matrix Multiply operations) should run in higher precision rather than FP8 and defining which statistics to log. Below, we outline how to structure the `config.yaml` file.\n",
+    "\n",
+    "## General Format\n",
+    "\n",
+    "A config file can have one or more sections, each containing settings for specific layers and features:\n",
+    "\n",
+    "```yaml\n",
+    "section_name_1:\n",
+    "  enabled: ...\n",
+    "  layers:\n",
+    "    # Specify layers here...\n",
+    "  transformer_engine:\n",
+    "    Feature1Name:\n",
+    "      enabled: ...\n",
+    "      # Feature details...\n",
+    "    Feature2Name:\n",
+    "      enabled: ...\n",
+    "      # Feature details...\n",
+    "\n",
+    "section_name_2:\n",
+    "  enabled: ...\n",
+    "  layers:\n",
+    "    # Specify layers here...\n",
+    "  transformer_engine:\n",
+    "    Feature1Name: # If feature has no namespace, then it is in the default namespace.\n",
```

Comment:
What do you mean by namespace here? I would think that the "transformer_engine:" above it is the namespace here, so why the comment? Also, this looks the same as in section 1, so why put the comment here and not there?